unity/src/project/SteamAudioUnity/Assets/Plugins/SteamAudio/Scripts/Runtime/SteamAudioSettings.cs

```diff
@@ -34,6 +34,7 @@ public class SteamAudioSettings : ScriptableObject
         public AudioEngineType audioEngine = AudioEngineType.Unity;
 
         [Header("HRTF Settings")]
+        public bool hrtfGlobalDisable = false;
```

Comment:
We're currently thinking through what the UI for this functionality should look like, if anything. I think for the moment we can just expose the relevant API functions, without adding this new setting or its corresponding custom inspector code.

Reply:
The reasons for why one might consider having this in the Steam Audio Settings are:
- Make people aware that this exists at all. API docs are more targeted at developers while non-developers like sound designers are more likely to look at inspectors. This would allow very small teams to simply build a version for headphones and a version for speakers requiring no coding.
- It's a setting related to HRTFs for which the Steam Audio Settings ScriptableObject already has a section for.
- It's a runtime setting similar to a lot of other settings on the Steam Audio Settings ScriptableObject in the sense that these are likely controllable by the end-user (like audio quality settings). The audio settings of a game using Steam Audio might consist of an Audio Quality Level dropdown control and a Headphone checkbox control. So having them all in the same place to set for developers makes it more consistent.
- It's similar to the "Perspective Correction" setting in so far as developers are likely to change this setting depending on whether the user starts their game in VR mode or in non-VR mode. The perspective correction setting is evaluated at runtime by the SteamAudioManager component and directly sent into the Steam Audio native plugin. As before, it might benefit consistency to have another HRTF related runtime setting in the same place.

In case we want to go with an API-only approach, I guess the most sensible thing to do would be to add a function to `SteamAudioManager.cs` looking like `public static void SetHRTFGlobalDisable` (or `SetHRTFDisable` for more consistency towards the requested changes).

Reply:
I think your points are valid, and the approach you've already implemented (i.e., providing a checkbox in the Steam Audio Settings to disable the HRTF) is the way to go. Feel free to address the remaining comments and commit to the PR, I'll do a quick final review before merging it in. Thanks for all the work, and thoughtful feedback!

Reply:
Alright, thanks a lot for reviewing this and generally Steam Audio!